src/components/layers-manager/LayersSwitcherContainer.js

```diff
@@ -4,9 +4,15 @@ import { toggleLayer, deleteLayer } from '../../modules/layers';
 
 const mapStateToProps = ({ layers, teams }) => {
   const selectedLayers = layers.selectedLayerIds.map((id) => layers.selectedLayers[id]);
-  const teamLayers = selectedLayers.filter((selectedLayer) => selectedLayer.attributes && selectedLayer.attributes.owner.type === 'TEAM')
-  const userLayers = selectedLayers.filter((selectedLayer) => selectedLayer.attributes && selectedLayer.attributes.owner.type === 'USER');
+  const publicLayers = selectedLayers.filter((selectedLayer) => selectedLayer.attributes && selectedLayer.attributes.isPublic);
```

Comment:
You seem to be doing this twice on both containers. Can you not do it once and pass it down through props. Much easier to scale.